### ISSUE TITLE
Reference Figures by Number

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#1637]
Potentially related to #702, but figures should be referenced by number (ie. Figure 1).

**Comment by smiths** [comment]
@samm82, good idea to reference related issues.  Can you again include a screen shot.  Also, we should point out that this issue is only relevant for the LaTex document.  LaTeX automatically generates numbers and within the document the figure is referenced by that number.  An alternative option would be to change the automatically generated LaTeX numbers to match the figures names (labels) that we are generating.  That sounds like an ugly idea to me though.  Ugly in the sense that it would be difficult to override the usual LaTeX behaviour and ugly in the appearance of the resulting document.

**Comment by samm82** [comment]
Are you saying that we shouldn't reference by Figure number in the HTML as well? Should we look into a way to automate it like it is done in the LaTeX?

HTML:
![html](https://user-images.githubusercontent.com/35857611/60294000-64113b00-98ee-11e9-9e01-e08c9c75a295.png)

**Comment by smiths** [comment]
Yes, @samm82, I'm saying that we don't need numbers in the html.  The html has a name and link and the figures do not have captions that mention numbers.  In LaTeX the figure captions have numbers that we then never use.  Similar to #1643, I would like the html version of the figures to have a refname that is displayed so that the hyperlink isn't the only way to disambiguate which is the intended figure.  Usually it will be clear, but if a link goes to a page with two figures, how does the reader know which figure is the one they should consult?  I guess the position on the page conveys this information, but the refname would be helpful.  It isn't a high priority, but it would mean better documentation.
